src/components/publishing/header/feature_header.tsx

```diff
@@ -288,7 +288,7 @@ const FeatureHeaderContainer = Div.extend`
 
 const sizeMeOptions = {
   refreshRate: sizeMeRefreshRate,
-  refreshMode: "debounce",
+  noPlaceholder: true,
```

Comment:
This helps address 2 issues:
1. No more flashing images
2. Inserts actual component elements instead of an empty div means we can calculate Read More truncation accurately on the `componentDidMount` lifecycle method